tools/ci_build/github/apple/c/assemble_c_pod_package.py

```diff
@@ -50,6 +52,8 @@ def main():
     # copy the necessary files to the staging directory
     framework_dir = args.framework_dir.resolve()
     shutil.copytree(framework_dir, staging_dir / framework_dir.name, dirs_exist_ok=True)
+    public_headers_dir = args.public_headers_dir.resolve()
+    shutil.copytree(public_headers_dir, staging_dir / public_headers_dir.name, dirs_exist_ok=True)
```

Comment:
>public_headers_dir [](http://example.com/codeflow?start=54&length=18)

is public_headers_dir.name required to be "Headers" now? that seems to be the case from the template file